### DISCUSSION TITLE
Better Note Browsing

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#800] [Feature Requests]
Currently browsing markdown files isn't ideal, one can follow links but each level you go down, makes it harder to exit, additionally it's not easy to know the exact file path you're on. (The only way is to try to rename the file and see it, which isn't ideal)

It would be nice to have some browser like navigation.

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Also, if you subscribe to the issue you will notified about its progress, and get a chance to see mockups and discussions about the feature.

**Comment by vHanda** [comment]
Trying out Dendron, it has shortcuts to go back to the note you were before. I think exposing this is quite important. Though I'm not sure how to it without adding too much clutter.

**Comment by alensiljak** [comment]
Would be great if some sort of breadcrumbs were available.
